packages/@lwc/template-compiler/src/__tests__/fixtures/static-content/deep-data-ref-only/expected.js

```diff
@@ -15,7 +15,7 @@ const stc3 = {
 function tmpl($api, $cmp, $slotset, $ctx) {
   const { sp: api_static_part, st: api_static_fragment } = $api;
   return [
-    api_static_fragment($fragment1(), 0, [
+    api_static_fragment($fragment1, 0, [
```

Comment:
This is mainly the change to all the template compiler fixtures, the invocation happens in the st function now so all the fixtures are updated like this.